vertx-web-graphql/src/main/java/io/vertx/ext/web/handler/graphql/GraphQLSocketHandler.java

```diff
@@ -0,0 +1,16 @@
+package io.vertx.ext.web.handler.graphql;
+
+import graphql.GraphQL;
+import io.vertx.core.Handler;
+import io.vertx.core.http.ServerWebSocket;
+import io.vertx.ext.web.handler.graphql.impl.GraphQLSocketHandlerImpl;
+
+public interface GraphQLSocketHandler extends Handler<ServerWebSocket> {
```

Comment:
For consistency with the other `vertx-web` utilities, it would be better to extend `Handler<RoutingContext>` and then implement the websocket handshake.